server/service/src/sync/program_schemas/encounters_arg_schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "EncounterStatus": {
+      "enum": ["SCHEDULED", "COMPLETED", "CANCELLED"],
```

Comment:
I've created a new database and these are the values for the encounter status:

![Screenshot 2023-02-13 at 11 58 28 AM](https://user-images.githubusercontent.com/9192912/218342494-bd4c5c62-1a59-4fe8-98b8-757b22bdeb39.png)

and if I try to print I get this

<img width="320" alt="Screenshot 2023-02-13 at 11 55 23 AM" src="https://user-images.githubusercontent.com/9192912/218342540-5545845d-f0f4-406c-a166-bc951b8ceda1.png">
 
because of this

```
"details": "DBError { msg: \"UNKNOWN\", extra: \"\\\"\\\\\\\"invalid input value for enum encounter_status: \\\\\\\\\\\\\\\"COMPLETED\\\\\\\\\\\\\\\"\\\\\\\"\\\"\" }"
```

Reply:
I've changed the `encounter_status` type in the database directly in order to test.
I'd suggest changing the type - that would make it consistent with the hiv_care and hiv_testing encounters too. Also, I like 'COMPLETED' better than 'DONE' 😉

Reply:
Yes Completed sounds better, was Carl's idea to rename it :)

Will fix it!

Reply:
Ok there where a couple of other issues as well, thought I ran the test but maybe not :(